AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1468,18 +1455,9 @@ - (void)__layout
   [self layoutDidFinish];
 }
 
+/// Needs to be called with lock held
```

Comment:
Is there a way to assert that the lock is held?  Might be good to start a naming scheme like _locked_measure... (just an idea, not required).